rules/Transform/Rector/Class_/AddAllowDynamicPropertiesAttributeRector.php

```diff
@@ -142,7 +142,7 @@ private function shouldSkip(Class_ $class): bool
         if ($this->transformOnNamespaces !== []) {
             $className = (string) $this->nodeNameResolver->getName($class);
             foreach ($this->transformOnNamespaces as $transformOnNamespace) {
-                if (! $this->nodeNameResolver->isStringName($className, $transformOnNamespace)) {
+                if (! $this->nodeNameResolver->matchesStringName($className, $transformOnNamespace)) {
```

Comment:
this requires a wildcard in the pattern now. should we call regular isStringName() beforehand to keep BC?

Reply:
Yes, this should check both :+1:

Reply:
done